misk/web/modules/log4j/src/core/app/index.ts

```diff
@@ -0,0 +1,39 @@
+import { createApp } from 'frint';
+import { createStore } from 'frint-store';
+const BrowserRouterService = require('frint-router/BrowserRouterService'); //require has to be used until it has typings
+
+import { RootContainer } from '../containers';
+import rootReducer from '../reducers';
+import { COLORS } from '../constants';
```

Comment:
I’ll want you to explain to me why some of these have braces!

Reply:
One of the idioms I like with directories of components or exports is having an `index.ts` file that imports all of the components in that directory and then export and object with all of them. Then instead of importing a longer path directly to the component file, the component can simply be unpacked using the `{ }` syntax from the directory path since it will then draw on `index.ts` to find the package. Cleaner syntax overall and forces registration of components in a single file and access of all components through that same `index.ts` file.

Example
`import { RootContainer } from '../containers';` instead of `import RootContainer from '../containers/RootContainer';`